channels/google-actions/src/main/kotlin/com/justai/jaicf/channel/googleactions/ActionsReactions.kt

```diff
@@ -47,29 +61,19 @@ class ActionsReactions(
         return ButtonsReaction.create(buttons.asList())
     }
 
-    override fun image(url: String): ImageReaction {
-        response.builder.add(Image().setUrl(url))
-        simpleResponse = null
+    override fun image(url: String) = image(url, null)
+
+    @Suppress("MemberVisibilityCanBePrivate")
+    fun image(url: String, accessibilityText: String?): ImageReaction {
+        val image = Image().setUrl(url)
+        accessibilityText?.let(image::setAccessibilityText)
+
+        response.builder.add(image)
         return ImageReaction.create(url)
     }
 
     fun endConversation() = response.builder.endConversation()
 
-    fun addSimpleResponse(displayText: String, ssml: String = displayText) {
```

Comment:
Deleting this public function breaks compatibility with previous versions. We should remain this signature.